CHANGES

```diff
@@ -30,6 +30,7 @@ Incompatible changes
 * html theme: Move a script tag for documentation_options.js in
   basic/layout.html to ``script_files`` variable
 * html theme: Move CSS tags in basic/layout.html to ``css_files`` variable
+* #8915: html theme: Emit a warning for sphinx_rtd_theme-0.2.4 or older
```

Comment:
Note: An entry_point for setuptools was added to sphinx_rtd_theme at 0.2.5 (see https://github.com/readthedocs/sphinx_rtd_theme/pull/425).
But Ubuntu-18.04 bundles v0.2.4
https://packages.ubuntu.com/bionic/python3-sphinx-rtd-theme

Reply:
I noticed v0.2.5 was not released. The next version of v0.2.4 is v0.3.0.
https://pypi.org/project/sphinx-rtd-theme/#history